app/js/services/preprocessing.js

```diff
@@ -33,14 +33,38 @@ angular
                 return formatTeamDisplayFilter(teamMap[element.relationships.team.data.id], false);
             }
         },
-				prettifyDate: function(date) {
-					if (date == undefined) { return; };
-					var monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
-					var year = parseInt(date.substring(0, 4));
-					var month = parseInt(date.substring(5, 7));
-					var day = parseInt(date.substring(8, 10));
-					return monthNames[month - 1] + " " + day + ", "+ year;
-				},
+        changeDate: function(element){
+            if(element.attributes != undefined){
+				element.attributes.created = this.convertTimeToEST(element.attributes.created);
+			}
+			if(element.attributes != undefined){
+				element.attributes.modified = this.convertTimeToEST(element.attributes.modified);
+            }
+            return element;
+        },
+        convertTimeToEST: function(time){
+            if(time == undefined){
+                return undefined;
+            }
+            var hour = parseInt(time.substring(11,13));
+            var minute = time.substring(14,16);
+            var night = 'AM';
+            if(hour > 12){
+                hour -= 12;
+                night = 'PM';
+            }
+            //return this.prettifyDate(time) + " " + hour + ":" + minute + night;
+            return moment(time).tz('America/New_York').format('LLLL');;
```

Comment:
can we do `October 1, 2014 12:00 AM (Wednesday)` instead of `Wednesday, October 1, 2014 12:00 AM`?